### DISCUSSION TITLE
可以在这里发帖或者参与讨论

### DISCUSSION THREAD

**Discussion by Tokinx** [Tokinx/Wing#3] [公告]
可以在这里发帖或者参与讨论，帖子类型很齐全

**Comment by zuorinet** [comment]
大佬，请教一下主题的友情链接和榜上有名要怎么设置

**Reply by Tokinx** [reply]
阅读wiki：https://github.com/Tokinx/Wing/wiki

**Reply by zuorinet** [reply]
榜上有名的这个页面是不是看恢复多少自动加上去的

**Reply by Tokinx** [reply]
> 榜上有名的这个页面是不是看恢复多少自动加上去的

是的，榜上有名是博客内一段时间内评论数量比较多的人会上榜

**Reply by zuorinet** [reply]
只要把模板选择读者排行就可以了吧

**Reply by Tokinx** [reply]
> 要把模板选择读者排行就可以了吧

是的

**Reply by zuorinet** [reply]
喜欢这款主题？

[抢先体验]
大佬，这个侧边栏要怎么添加

**Reply by Tokinx** [reply]
> 喜欢这款主题？
> 
> [抢先体验] 大佬，这个侧边栏要怎么添加

html小组建

**Comment by Dick-Williams** [comment]
前台发表随笔的时候会报错
![Screenshot_20220512_154026_mark via](https://user-images.githubusercontent.com/83115890/168018811-662fbd2d-d98e-40e2-98e9-6fbabaee960f.jpg)

**Reply by Dick-Williams** [reply]
是我设置的问题吗

**Reply by Tokinx** [reply]
需要设置固定连接，开启伪静态

**Reply by Dick-Williams** [reply]
> 需要设置固定连接，开启美好的风景

啊这 所以应该在哪里设置呢

**Reply by oxn** [reply]
设置固定链接，一般情况下在【仪表盘】-【设置】-【固定链接】设置之后wordpress会修改【.htaccess】文件，如果修改失败，你需要自己修改【.htaccess】文件。
`# BEGIN WordPress
# 在“BEGIN WordPress”与“END WordPress”之间的指令（行）是
# 动态生成的，只应被WordPress过滤器修改。
# 任何对标记之间的指令的修改都会被覆盖。
<IfModule mod_rewrite.c>
RewriteEngine On
RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]
RewriteBase /
RewriteRule ^index\.php$ - [L]
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule . /index.php [L]
</IfModule>

# END WordPress
`

**Reply by oxn** [reply]
![h](https://user-images.githubusercontent.com/3295116/169973548-0c22389f-a945-4b42-8f66-0fffc283938b.png)

**Comment by oxn** [comment]
大佬，我发现的我的友情链接界面。没有那个中部的显示个人资料的东西，不光是友情链接，所有页面中都没有，而文章中是有的。我看你的页面中都有，不知道原因，不知道其他朋友有没有遇见过这个问题。在线等谢谢了。

**Reply by Tokinx** [reply]
应该是你没有选择模板，你编辑一下页面，页面模板选择“友情链接”？

**Reply by oxn** [reply]
选择模板了，大佬。

**Comment by oxn** [comment]
![友情链接](https://user-images.githubusercontent.com/3295116/169774742-4585a647-6ba4-4d77-8eb0-88837d47f5dc.png)
![友情链接2](https://user-images.githubusercontent.com/3295116/169774748-e69b310c-821f-4f83-b4f1-890d4020ddf0.png)

**Reply by Tokinx** [reply]
似乎是因为关闭了评论

**Reply by oxn** [reply]
收到，谢谢

**Comment by oxn** [comment]
大佬，今天发现360浏览器使用兼容模式，你的主页是空白显示的，极速模式谷歌内核的没有问题。有时间您测试一下试试，谢谢了。
![360](https://user-images.githubusercontent.com/3295116/169962766-7ee16793-37e1-482c-95c2-ca70109dcf8d.png)
![3601](https://user-images.githubusercontent.com/3295116/169962771-de104cb6-0dec-4506-8660-a58b26a7e744.png)

**Reply by Tokinx** [reply]
因为我手里没有IE浏览器，暂时无法测试到，IE应该是不兼容的（360的兼容性模式就是IE模式）

**Comment by oxn** [comment]
还有个问题，大佬，就是如果有置顶的文章，那么首页显示会多一个，其他页没有问题，这个是不是wordpress自身问题？无法解决吧？

**Reply by Tokinx** [reply]
提issues吧，我没有过滤掉置顶文章导致的。
另外有问题只在github上提就好了，不用在博客评论。主题还没有正式发布，只是预览版提前放出来体验而已，有BUG很正常

**Reply by oxn** [reply]
好的，大佬，我访问github总出问题，有时无法访问。以后就在这里和您汇报使用感受。

**Comment by oxn** [comment]
![头像](https://user-images.githubusercontent.com/3295116/169971179-72b9a6a1-d60b-4e60-9a78-6ef382b89079.png)
我设置的是https://gravatar.loli.net/的评论区的都能正常显示，出了博主意外都显示正常，但是博主的头像就是在评论自己的文章或者页面时，对话框左边的显示不正常。地址成了https://secure.gravatar.com/，不知道您的有没有这样的问题。

**Reply by Tokinx** [reply]
更新主题即可

**Reply by oxn** [reply]
![新版本](https://user-images.githubusercontent.com/3295116/169988340-9eec2bc9-bc85-4923-b226-759373bb7a52.png)
新版本还是有问题。大佬。

**Comment by oxn** [comment]
还有就是设置文章置顶后，文章首页会多显示，这个能想办法避免吗？还有就是文章页能设置不显示缩微图，只显示文字吗？

**Reply by oxn** [reply]
大佬刚刚看了你的主页，挺好看，是文字与图一起显示的，漂亮。

**Comment by oxn** [comment]
![卡顿](https://user-images.githubusercontent.com/3295116/170630969-8a957d3a-72b0-4ae9-a815-46426b9ea904.png)
![卡顿2](https://user-images.githubusercontent.com/3295116/170630979-48268611-8301-4076-a718-e370099772f1.png)
大佬，你看下我每次跳到后台都这样。

**Reply by Tokinx** [reply]
进后台要新窗口打开

**Reply by oxn** [reply]
是我服务器的问题吗？大佬

**Reply by Tokinx** [reply]
不是服务器问题，是主题的PJAX不支持

**Reply by oxn** [reply]
明白了，谢谢大佬。

**Reply by oxn** [reply]
大佬，我发现笔记中的发的图片在移动端可以点击打开放大看，而在电脑端点击没有反应，我的浏览器是谷歌浏览器。

**Reply by oxn** [reply]
纠正下。说错了，大佬是360浏览器极速模式，谷歌可以打开。。。。

**Comment by oxn** [comment]
1.360的极速浏览器X功能正常。
2.360安全浏览器有问题，极速模式照片点击没有反应，F12控制台显示
Uncaught (in promise) TypeError: Cannot read property 'setLoading' of undefined
    at modules.js?ver=0.7.7:467
    at <anonymous>
3.360安全浏览器兼容模式依然空白显示。

**Reply by Tokinx** [reply]
主题不支持IE，兼容性模式是IE内核访问所以无法显示内容。
缩略图报错的问题我需要再看一下具体什么问题

**Reply by oxn** [reply]
大佬我这访问github总出问题，有时发完了一个问题想删除，就进不来了，或者编辑完问题有时就发不出去了，比较尴尬。大佬，希望您方便可以加入JS代码自定义。方便添加统计代码，不然用统计插件消耗数据库空间，而且那几个统计插件感觉不友好。谢谢大佬，辛苦了。

**Comment by oxn** [comment]
![360安全浏览器点击图片](https://user-images.githubusercontent.com/3295116/170898967-e3675793-66dd-47b3-bb46-9113347d3abf.png)

**Comment by oxn** [comment]
已使用主题的ICP备案，公安备案，js，及尾部信息功能，大赞！

**Comment by oxn** [comment]
希望下个版本可以推出个打赏功能。。。特别期待。

**Comment by oxn** [comment]
我感觉明亮模式备案文字看不清

**Comment by luanluke** [comment]
大佬，我已经部署好了wing模板，各方面都设置好了。现在唯一的疑惑就是，我的博客整体的字体和页面的比例都比你的大，显得不精致，请问怎样调小？

**Comment by lo-vemla** [comment]
大佬，设置友情链接的时候没看到有设置的按钮，显示是这样的，不知道怎么处理了，请指教谢谢~
![1](https://user-images.githubusercontent.com/120256787/206855892-874defe8-3dd2-4407-bbce-bb2fde35e2aa.png)

**Reply by LingJueYa** [reply]
在后台页面设置开启可以评论

**Comment by yongbin1999** [comment]
![image](https://github.com/Tokinx/Wing/assets/47911980/5d169d3c-bf06-48aa-9806-0b213c796cbb)
![image](https://github.com/Tokinx/Wing/assets/47911980/b8008703-bfba-483e-8a02-a57b1765581c)
大佬，我已经开启了评论，模板也设置了友情链接，但还是没有出现个人信息和设置，翻阅了所有讨论，还是没有找到原因
